packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1264,7 +1346,7 @@ export default class Meeting extends StatelessWebexPlugin {
             // We need to check what should be the actual direction of media
             sendAudio: this.mediaProperties.mediaDirection?.sendAudio && !this.audio?.isMuted(),
             sendVideo: this.mediaProperties.mediaDirection?.sendVideo && !this.video?.isMuted(),
-            sendShare: this.mediaProperties.mediaDirection?.sendShare && this.isSharing,
```

Comment:
check the console logs to see if there are no warning or errors on stats module

Reply:
I've checked - I haven't seen any warnings or errors in any of the testing I did